internal/controller/volumereplicationgroup_controller.go

```diff
@@ -548,6 +556,10 @@ func (v *VRGInstance) processVRG() ctrl.Result {
 		return v.invalid(err, "Failed to process list of PVCs to protect", true)
 	}
 
+	if err := v.updatePVCListForCG(); err != nil {
```

Comment:
(nit) rename this function to say something like `labelPVCsForCG`. With the other updatePVCList` function just above, it is misleading in intent (as it really does not update the PVC lists as such).

Reply:
Fixed